db/seeds.rb

```diff
@@ -18,3 +18,11 @@
                activated_at: Time.zone.now
               )
 end
+users = User.all
+user = users.first
+following = users[2..20]
+followers = users[3..15]
+following.each{|followed|
+  user.follow(followed)}
+followers.each{|follower|
+  follower.follow(user)}
```

Comment:
```suggestion
following.each{|followed| user.follow(followed)}
followers.each{|follower| follower.follow(user)}
```
Nếu xuống dòng thì phải dùng do .. end em nha. Còn trên 1 dòng thì {...}